platforms/macos/src/node.rs

```diff
@@ -528,6 +787,25 @@ declare_class!(
                 if selector == sel!(accessibilityPerformDecrement) {
                     return node.supports_decrement();
                 }
+                if selector == sel!(accessibilityNumberOfCharacters)
+                    || selector == sel!(accessibilitySelectedText)
+                    || selector == sel!(accessibilitySelectedTextRange)
+                    || selector == sel!(accessibilityInsertionPointLineNumber)
+                    || selector == sel!(accessibilityRangeForLine:)
+                    || selector == sel!(accessibilityRangeForPosition:)
+                    || selector == sel!(accessibilityStringForRange:)
+                    || selector == sel!(accessibilityAttributedStringForRange:)
+                    || selector == sel!(accessibilityFrameForRange:)
+                    || selector == sel!(accessibilityLineForIndex:)
+                    || selector == sel!(accessibilityRangeForIndex:)
+                    // This adapter doesn't yet actually support the SetValue
+                    // action, but we must claim to support it in order
+                    // to get the expected VoiceOver behavior in edits.
+                    || selector == sel!(setAccessibilityValue:)
```

Comment:
Interesting that this is necessary, thanks for documenting it.

Have you tested what happens if you try to set the value via something like Accessibility Inspector?